backend/lcfs/web/api/fuel_code/repo.py

```diff
@@ -217,13 +228,13 @@ async def get_fuel_type_by_id(self, fuel_type_id: int) -> FuelType:
     @repo_handler
     async def get_fuel_categories(self) -> List[FuelCategory]:
         """Get all fuel category options"""
-        return (await self.db.execute(select(FuelCategory))).scalars().all()
+        return (await self.db.execute(select(FuelCategory))).unique().scalars().all()
```

Comment:
same here, FuelCategory only has 3 rows, why do you need unique?

Reply:
Thank you! I removed eager-loaded relationship.